docs/modules/commons/pages/cli.adoc

```diff
@@ -0,0 +1,290 @@
+= CLI
```

Comment:
I think we also need to add:
IDE tasks
---------
cleanEclipse - Cleans all Eclipse files.
cleanIdea - Cleans IDEA project files (IML, IPR)
eclipse - Generates all Eclipse files.
idea - Generates IDEA project files (IML, IPR, IWS)
openIdea - Opens the IDEA project

Build tasks
-----------
build - Assembles and tests this project.
clean - Deletes the build directory.

Reply:
Feel free to create a ticket

Reply:
https://github.com/vividus-framework/vividus/issues/2826